packages/website/docs/DocSearch-program.md

```diff
@@ -78,6 +78,18 @@ The information of the initially applied project is used everywhere when we depl
 
 This allow us to easily scope issues when reaching out for support.
 
+## Support
+
+We do not provide Algolia support for DocSearch, but for the products that are used by the program.
+
+Please make sure to **first read the documentation (or [the Algolia one](https://www.algolia.com/doc/)) before reaching out** and take a look at [the Algolia academy](https://academy.algolia.com/trainings).
+
+For issues related to:
```

Comment:
`bug report` might avoid confusion about what is an issue

Reply:
I've removed this line with the suggestion you just made, it makes more sense